etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -267,11 +275,50 @@ def load(cls, path: pathlib.Path) -> "TS2VecEmbeddingModel":
         path:
             Path to load object from.
 
+            - if `path` is not None and `model_name` is None, load the local model from `path`.
+            - if `path` is None and `model_name` is not None, save the external `model_name` model to the etna folder in the home directory and load it.
+            If `path` exists, external model will not be loaded.
```

Comment:
This line doesn't make much sense. We have `path=None` here. 

I also don't agree that "external model will not be loaded". It will be loaded, but it won't be downloaded from the internet.